tasks.rb

```diff
@@ -493,6 +493,8 @@ def set_measure_argument_values(hpxml_file, args)
 
   if ['base.xml'].include? hpxml_file
     args['simulation_control_timestep'] = 60
+    args['site_iecc_zone'] = '5B'
+    args['site_state_code'] = 'CO'
```

Comment:
Does this actually change anything? And why don't I see updated HPXML files with these values?

Reply:
It doesn't change anything because the `BuildResidentialHPXML` measure was already filling these in. Calling these arguments out in `tasks.rb` keeps them populated in the sample files.

Reply:
Ah, that makes sense. Thanks.